src/bin/parser_test.rs

```diff
@@ -29,6 +30,11 @@ fn main() -> io::Result<()> {
     let default_dir = "./html5lib-tests";
     let dir = env::args().nth(1).unwrap_or(default_dir.to_string());
 
+    if !Path::new(&dir).exists() {
+        println!("Missing 'html5lib-tests'. It is no located at '{dir}' \nIt should be cloned first from here https://github.com/html5lib/html5lib-tests");
```

Comment:
Hey there, I noticed a small typo

```suggestion
        println!("Missing 'html5lib-tests'. It is not located at '{dir}' \nIt should be cloned first from here https://github.com/html5lib/html5lib-tests");
```